minicluster/src/main/java/org/apache/accumulo/minicluster/MiniAccumuloConfig.java

```diff
@@ -108,6 +108,19 @@ public MiniAccumuloConfig setZooKeeperStartupTime(long zooKeeperStartupTime) {
   }
 
   /**
+   * Configure an existing ZooKeeper instance to use. Calling this method is optional. If not set, a new ZooKeeper instance is created.
+   *
+   * @param existingZooKeepers
+   *          Connection string for a already-running ZooKeeper instance
+   *
+   * @since 1.7.0
+   */
+  public MiniAccumuloConfig setExistingZooKeepers(String existingZooKeepers) {
```

Comment:
> I am really curious about the performance impact.

Looking at some [Jenkins logs](https://secure.penguinsinabox.com/jenkins/job/Accumulo-Master-Integration-Tests/ws/test/target/mini-tests/org.apache.accumulo.test.ConditionalWriterIT_testBasic/logs/), it looks like starting ZK and waiting for it to come up takes ~3seconds (time on first ZK log msg compared to time on first tserver log msg). More quantitative measurements would be cool to collect as well.